pkg/collector/corechecks/oracle/config/README.TXT

```diff
@@ -0,0 +1,30 @@
+Currently supported configurations
+----------------------------------
+- multi-tenant (no legacy architecture)
+- single node instance (no RAC and Exadata)
+- requires SYSDBA in CDB for setup (no RDS)
+- tested with Oracle releases: 19c and 21c
+
+Database
+--------
+Connect as sysdba to the monitored CDB database and execute the following sql script in the monitored database:
+
+https://github.com/DataDog/datadog-agent/blob/nenad.noveljic/dbm-oracle-beta-7.43.x/pkg/collector/corechecks/oracle/sql/setup.sql
+
+The script will ask for password and create the monitoring account. If the account already exists, the CREATE USER command will fail, but the other commands will execute successfully.
+
+Agent
+-----
+On the server where the agent is running, install the agent version dbm-oracle-beta-0.17-1.
+
+Create new configuration directory for oracle-dbm:
+
+CONF_D=/etc/datadog-agent/conf.d
+mkdir $CONF_D/oracle-dbm.d
```

Comment:
Is it intentional that you didn't explicitly add this directory and rather we're asking the user to manually create it?

Reply:
It wasn't until yesterday that I learned how the creation of the new files is handled automatically. I wanted to create a new PR for that immediately after this PR is merged.

Reply:
Changed it now

Reply:
Changed the [README.TXT](https://github.com/DataDog/datadog-agent/blob/nenad.noveljic/dbm-oracle/pkg/collector/corechecks/oracle/config/README.TXT) now: . I also added `oracle-dbm.d`